eng/common/scripts/Create-APIReview.ps1

```diff
@@ -92,7 +97,6 @@ foreach ($pkgName in $responses.Keys)
         if (-Not (Test-Path $pkgPropPath))
         {
             Write-Host " Package property file path $($pkgPropPath) is invalid."
-            $FoundFailure = $True
```

Comment:
Why was this removed?

Reply:
Package property config file is created only for new SDK type to avoid conflict between track1 and track2 packages within different group(java). This is per our discussion earlier to keep it simple. Pipeline step is marked as failed if package property file is missing for any package which will cause an issue when we enable this failure as hard error in the near future.

Reply:
OK I think I understand the logic now. I will say however this has the potential for track 2 libraries that for whatever reason that fail to publish a package info file from requiring approval.